docs/development.md

```
@@ -1,462 +1,175 @@
 
-# 4. クエリ層実装
-# app/queries/ にTanStack Query統合
+ファイルを編集すると、ブラウザが自動で更新されます。これを**ホットリロード**と呼びます。
 
-# 5. コンポーザブル実装
-# app/composables/ に統合ロジック
+#### よく編集するファイル
 
-# 6. コンポーネント実装
-# app/components/ にUI実装
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)